model-optimizer/extensions/front/kaldi/replace_lstm_nonlinearity.py

```diff
@@ -27,12 +27,26 @@ def run_before(self):
         return [FullyConnectedDecomposer]
 
     def replace_op(self, graph: Graph, node: Node):
-        # split input to (i_part, f_part, c_part, o_part, ct_1)
         node_name = node.soft_get('name', node.id)
+        # check if we have dropout
+        input_port = node.in_port(0)
+        if node['use_dropout']:
```

Comment:
please use `node.has_and_set('use_dropout')`

Reply:
why? We set attribute in init of LstmNonlinearity operation, it can't be created without set use_dropout

Reply:
`node.has_and_set('use_dropout')` is short equivalent to `'use_dropout' in node and node['use_dropout']==True`